app/recruitment/admin.py

```
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Ruff (0.12.2)

`django.contrib.admin` imported but unused

Remove unused import: `django.contrib.admin`

(F401)